skimage/restoration/_denoise.py

```diff
@@ -277,17 +282,38 @@ def denoise_tv_bregman(image, weight, max_iter=100, eps=1e-3, isotropic=True):
 
     """
     image = np.atleast_3d(img_as_float(image))
-    image = np.ascontiguousarray(image)
 
-    rows = image.shape[0]
-    cols = image.shape[1]
-    dims = image.shape[2]
-
-    shape_ext = (rows + 2, cols + 2, dims)
+    shape_ext = np.asarray(image.shape)
+    shape_ext[0:2] += 2
 
     out = np.zeros(shape_ext, image.dtype)
-    _denoise_tv_bregman(image, image.dtype.type(weight), max_iter, eps,
-                        isotropic, out)
+
+    if multichannel:
+        for c in range(image.shape[-1]):
+            if np.ndim(image) == 3:
+                channel_in = np.ascontiguousarray(image[..., c, None])
```

Comment:
why do you use `None` at the end?

Reply:
None adds a extra dimension after slicing.